lib/utils.js

```diff
@@ -35,18 +46,45 @@ module.exports.zoomsBySize = function(filepath, extent, callback) {
       avg = stats.size / tiles;
 
       if (avg < 1000) max = z;
```

Comment:
Perhaps this `1000 tiles` metric is also something we could revisit, revisiting our sense of how many tiles is too many tiles